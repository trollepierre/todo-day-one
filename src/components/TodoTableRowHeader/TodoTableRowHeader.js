export const TodoTableRowHeader = () => (
  <tr>
    <th>title</th>
    <th>Type</th>
    <th>Created At</th>
    <th>isDone?</th>
  </tr>
)
