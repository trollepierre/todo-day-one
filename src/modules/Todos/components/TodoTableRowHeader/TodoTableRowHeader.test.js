import { render, } from '@testing-library/react'
import { TodoTableRowHeader } from './TodoTableRowHeader'

describe('TodoTableRowHeader', () => {
  it('should match snapshot',  () => {
    const { container } = render(
      <table><tbody>
        <TodoTableRowHeader/>
      </tbody></table>
    )
    expect(container).toMatchSnapshot()
  })
})
