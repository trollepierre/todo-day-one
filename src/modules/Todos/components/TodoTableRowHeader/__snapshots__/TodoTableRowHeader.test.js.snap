// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoTableRowHeader should match snapshot 1`] = `
<div>
  <table>
    <tbody>
      <tr>
        <th>
          title
        </th>
        <th>
          Type
        </th>
        <th>
          Created At
        </th>
        <th>
          isDone?
        </th>
      </tr>
    </tbody>
  </table>
</div>
`;
